integration_tests/test_domestic_migration.py

```diff
@@ -1,23 +1,91 @@
+import os
+from pathlib import Path
+
 import numpy as np
 import pandas as pd
 import pytest
+import yaml
+from vivarium.framework.utilities import from_yearly
 
 from vivarium_census_prl_synth_pop.constants import data_values, metadata, paths
 
+from .conftest import FuzzyChecker
 
-def test_individuals_move(simulants_on_adjacent_timesteps):
-    for before, after in simulants_on_adjacent_timesteps:
+
+@pytest.fixture(scope="module")
+def target_migration_rates(sim):
+    assert (
+        metadata.UNITED_STATES_LOCATIONS == []
+    ), "Integration tests do not support subsets by US state"
+
+    with open(
+        Path(os.path.dirname(__file__)) / "v_and_v_inputs/domestic_migration.yaml"
+    ) as f:
+        targets = yaml.safe_load(f)
+
+    targets["individual_migration_rate_per_year"]["total"] = sum(
+        [v for v in targets["individual_migration_rate_per_year"].values()]
+    )
+
+    targets["household"] = from_yearly(
+        targets["household_migration_rate_per_year"],
+        pd.Timedelta(days=sim.configuration.time.step_size),
+    )
+
+    targets["individual"] = {}
+    for k in targets["individual_migration_rate_per_year"].keys():
+        targets["individual"][k] = from_yearly(
+            targets["individual_migration_rate_per_year"][k],
+            pd.Timedelta(days=sim.configuration.time.step_size),
+        )
+
+    return targets
+
+
+def test_individuals_move(
+    simulants_on_adjacent_timesteps, fuzzy_checker: FuzzyChecker, target_migration_rates
+):
+    all_time_individual_movers = []
+
+    for time_steps, (before, after) in enumerate(simulants_on_adjacent_timesteps):
         individual_movers = before["household_id"] != after["household_id"]
-        assert individual_movers.any()
+        fuzzy_checker.fuzzy_assert_proportion(
+            "Individual migration rate",
+            individual_movers,
+            # We say it could vary by up to 1% per timestep because of demographic
+            # change over the sim
+            target_value_lb=target_migration_rates["individual"]["total"]
+            * pow(0.99, time_steps),
+            target_value_ub=target_migration_rates["individual"]["total"]
+            * pow(1.01, time_steps),
+            name_addl=f"Time step {time_steps}",
+        )
         assert (
             before[individual_movers]["household_details.address_id"]
             != after[individual_movers]["household_details.address_id"]
         ).all()
         check_po_box_collisions(after, before, individual_movers)
+        all_time_individual_movers.append(individual_movers)
+
+    all_time_individual_movers = pd.concat(all_time_individual_movers, ignore_index=True)
+    fuzzy_checker.fuzzy_assert_proportion(
+        "Individual migration rate",
+        all_time_individual_movers,
+        # We say it could vary by up to 10% because of demographic
+        # change over the sim
+        target_value_lb=target_migration_rates["individual"]["total"] * 0.9,
```

Comment:
For simplicity, let's aggregate this from the input drift per unit time, assuming each timestep has equal population size (roughly true). Create a function to do this.